site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/pfi/PfiOnboardingTest.kt

```diff
@@ -31,7 +31,7 @@ class PfiOnboardingTest {
         services = listOf(serviceToAdd),
     )
 
-    val did = DidDht.create(InMemoryKeyManager(), options)
+    val pfiDid = DidDht.create(InMemoryKeyManager(), options)
     // :snippet-end:
 
     assertEquals("PFI", did.didDocument?.services?.get(0)?.type, "DID should start with 'did:key'")
```

Comment:
how is this passing when it's using `did` and you renamed it to `pfiDid`? 

```suggestion
    assertEquals("PFI", pfiDid.didDocument?.services?.get(0)?.type, "DID has PFI service type")
```

Reply:
ah it's not passing

https://github.com/TBD54566975/developer.tbd.website/actions/runs/7839445505/job/21392477069?pr=1209#step:10:13855

Reply:
Fixed